File: data_structures/array/array.go
package main

import "fmt"

func main() {
	fmt.Println("arrays")

	var numbers = [3]int{1, 2, 3}
	fmt.Println(numbers[0], numbers[len(numbers)-1])
	fmt.Println(len(numbers), cap(numbers))

	numbers[1] = 0
	fmt.Println(numbers)
}

File: data_structures/array/array.rs
fn main() {
    println!("arrays");

    let mut arr: [u8; 3] = [1, 2, 3];
    println!("{:?}", arr);

    println!("{} {}", arr[0], arr[arr.len() - 1]);

    arr[1] = 0;
    println!("{:?}", arr);
}
